JiraPS/Public/Remove-JiraVersion.ps1

```diff
@@ -10,60 +11,75 @@
     .EXAMPLE
        Remove-JiraVersion -ID '66596'
        This example removes the Version given.
-    .INPUTS
-       This function does not accept pipeline input pending class creation.
+     .INPUTS
+        [PSJira.Versions]
     .OUTPUTS
        This Function outputs no results
     .NOTES
        This function requires either the -Credential parameter to be passed or a persistent JIRA session. See New-JiraSession for more details.  If neither are supplied, this function will run with anonymous access to JIRA.
     #>
-    [CmdletBinding(DefaultParameterSetName = 'Project')]
+    [CmdletBinding(DefaultParameterSetName = 'VersionID')]
```

Comment:
I would like parametersets to be called `by*`.
I believe it makes for easier reading in the help.